tracing-subscriber/src/subscribe/mod.rs

```diff
@@ -1475,6 +1475,26 @@ pub struct Identity {
 
 // === impl Subscribe ===
 
+#[derive(Clone, Copy)]
+pub(crate) struct NoneLayerMarker;
+pub(crate) static NONE_LAYER_MARKER: NoneLayerMarker = NoneLayerMarker;
+
+/// Does a type implementing `Subscriber` contain any per-subscriber filters?
```

Comment:
this comment is incorrect, looks like it was copied from the per-subscriber filter detection function?